bumble/hfp.py

```diff
@@ -390,7 +391,7 @@ async def execute_command(
                 )
                 if result.code == 'OK':
                     if response_type == AtResponseType.SINGLE and len(responses) != 1:
-                        raise ProtocolError("NO ANSWER")
+                        raise InvalidStateError("NO ANSWER")
```

Comment:
I think `ProtocolError` is more appropriate here. An `InvalidStateError` is usually more representative of a condition like trying to close a stream that's not open. a `ProtocolError` reflects an error in an exchange between two entities, where the syntax or sequence of messages isn't what's expected or allowed, which seems to be the case here.

Reply:
I extend BaseError to show non-digital errors.